transcom/mymove
# Add the milmove_trace_id to client logs

## Summary

With https://github.com/transcom/mymove/pull/11183 merged, we get client logs uploaded to CloudWatch.

This PR adds the `milmove_trace_id`, if possible, to those logs if an API request fails. This makes it possible to then correlate the client errors with the server requests.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### How to test

1. `make db_dev_truncate`
1. `./scripts/server-dev --standalone`
1. Go to <http://officelocal:8080>
1. Click `Local Sign In`
1. Create a new Services Counselor User
1. Open the developer tools so you can see the console 
1. Go to <http://officelocal:8080/counseling/moves/XXXXXX/details> 
1. See in the console that the `milmove_trace_id` is now logged

### Frontend
